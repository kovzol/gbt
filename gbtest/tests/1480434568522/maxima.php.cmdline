#!/bin/sh
cd tests/1480434568522
cat maxima.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/maxima > maxima.php.out 2>maxima.php.err