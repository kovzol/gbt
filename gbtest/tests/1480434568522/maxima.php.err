0.26user 0.00system 0:00.28elapsed 96%CPU (0avgtext+0avgdata 96128maxresident)k
0inputs+480outputs (0major+4507minor)pagefaults 0swaps
