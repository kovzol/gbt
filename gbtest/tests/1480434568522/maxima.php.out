
Maxima 5.37.2 http://maxima.sourceforge.net
using Lisp GNU Common Lisp (GCL) GCL 2.6.12
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1) (%o3) [v11, v9, v12 - 1, v10, v7 - 1, v6, v8, v5, 
     2       2     2
(- v2 ) + v19  - v1  + 2 v1 - 1, v4, v3, 1 - 2 v15, 
       4            3       2   2          2     2       2
(- 3 v2 ) + 4 v16 v2  - 6 v1  v2  + 6 v1 v2  - v2  + 4 v1  v16 v2 - 4 v1 v16 v2
       4       3       2     2     2     2           2
 - 3 v1  + 6 v1  - 3 v1 , v20  - v2  - v1 , 3 - 4 v16 , 
   2     2                2
v21  - v2  + 2 v16 v2 - v1  + v1 - 1]
(%i4) 